db/migrate/20221209093842_add_email_on_donation_notification_to_communication_preferences.rb

```diff
@@ -0,0 +1,5 @@
+class AddEmailOnDonationNotificationToCommunicationPreferences < ActiveRecord::Migration[7.0]
+  def change
+    add_column :user_communication_preferences, :email_on_donation_notification, :boolean, default: true, null: false
```

Comment:
```suggestion
    add_column :user_communication_preferences, :email_about_fundraising_campaigns, :boolean, default: true, null: false
```

We're not going to use a notification to send these, so let's name it like this instead.